src/en/installconfig-rack.md

```diff
@@ -21,8 +22,8 @@ controller must be able to:
 
 ## Install a rack controller
 
-When a rack controller is installed on the same system as the region controller
-it will be registered to MAAS automatically. Otherwise it will need to be done
+When a rack controller is installed on the same host as the region API server
+it will register itself to it automatically. Otherwise it will need to be done
```

Comment:
Also, 'Otherwise it will need...' Can't we just say 'Otherwise this will need to be done manually' 

Q: How do we do this manually - is it explained below?